includes/class-wc-gateway-stripe.php

```diff
@@ -1266,4 +1266,48 @@ public function settings_api_sanitized_fields( $settings ) {
 		}
 		return $settings;
 	}
+
+	/**
+	 * This is overloading the title type so the oauth url is only fetched if we are on the settings page.
+	 *
+	 * @param string $key Field key.
+	 * @param array  $data Field data.
```

Comment:
should we add a `@since` tag here?

Reply:
I find `@since` are most appropriate for method that are designed to be used by third parties. In this case I wouldn't expect a third party to use `generate_stripe_account_keys_html()` and would make it private if I could.